src/logger.cr

```diff
@@ -0,0 +1,72 @@
+class Logger
+  property :level, :progname, :formatter
+
+  UNKNOWN = 5
+  FATAL = 4
+  ERROR = 3
+  WARN = 2
+  INFO = 1
+  DEBUG = 0
+
+  SEV_LABEL = %w(DEBUG INFO WARN ERROR FATAL ANY)
+
+  DEFAULT_FORMATTER = ->(severity : String, datetime : Time, progname : String, message : String) {
+    "#{severity[0]}, [#{datetime} ##{Process.pid}] #{severity.rjust(5)} -- #{progname}: #{message}"
+  }
```

Comment:
The downside is that I can't fake a Function with a class:

```
class MyFormatter
  def call severity, datetime, progname, message
  end
end
```

Or I could, but I would need to know about the `->Formatter.new.call` syntax.